src/mission_start.cpp

```diff
@@ -142,6 +142,21 @@ void mission_start::infect_npc( mission *miss )
     p->guard_current_pos();
 }
 
+void mission_start::asthmatic_npc( mission *miss )
```

Comment:
This could be generalized for all "give me drugs or I die" missions - if you changed it to remove the mission item type instead of hardcoded inhaler, it would be usable for things like antifungal drugs, antiparasitic drugs, insulin or whatever.